focus-ios/Blockzilla/UIComponents/UIConstants.swift

```diff
@@ -213,7 +211,14 @@ struct UIConstants {
             value: "This helps us diagnose and fix issues with the browser.",
             comment: "Description associated with the Crash Reports toggle on settings screen"
         )
-
+        static let labelDailyUsagePing = NSLocalizedString(
```

Comment:
Tagging @Delphine for these strings. I missed the deadline yesterday for string freeze.